components/libconfig/libconfig.p5m

```diff
@@ -0,0 +1,60 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+# Copyright (c) 2011, 2013, Oracle and/or its affiliates. All rights reserved.
+#
+
+<transform file path=usr.*/man/.+ -> default mangler.man.stability uncommitted>
+set name=pkg.fmri \
+    value=pkg:/library/libconfig@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="C/C++ library for processing configuration files"
+set name=pkg.description \
+    value="C/C++ library for processing configuration files"
+set name=com.oracle.info.description value="C/C++ library for processing configuration files"
+set name=info.classification \
+    value=org.opensolaris.category.2008:System/Libraries
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=org.opensolaris.arc-caseid value=PSARC/2009/118 value=PSARC/2013/044
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+dir path=usr/share/info
+file path=usr/share/info/libconfig.info group=bin mode=0444 owner=root \
+restart_fmri=svc:/application/texinfo-update:default facet.doc.info=true 
+file path=/usr/include/libconfig.h
+file path=usr/include/libconfig.h++
+file path=usr/lib/libconfig.la
```

Comment:
When I do this, it generates a new file with the following header :
# A full copy of the text of the CDDL should have accompanied this
# source. A copy of the CDDL is also available via the Internet at
# http://www.illumos.org/license/CDDL.

Like the header says, should I include the 377 lines license from the website or leave the text that suggests that something have been done improperly ?

Best regards.
Ben

----- Mail original -----
De: "Alexander Pyhalov" <notifications@github.com>
À: "OpenIndiana/oi-userland" <oi-userland@noreply.github.com>
Cc: "benf33" <bentahyr@chez.com>
Envoyé: Mardi 16 Février 2016 20:31:32
Objet: Re: [oi-userland] sslh + libconfig as a req (#1701)



In components/libconfig/libconfig.p5m : > +set name=pkg.summary value="C/C++ library for processing configuration files"
> +set name=pkg.description \
> +    value="C/C++ library for processing configuration files"
> +set name=com.oracle.info.description value="C/C++ library for processing configuration files"
> +set name=info.classification \
> +    value=org.opensolaris.category.2008:System/Libraries
> +set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
> +set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
> +set name=org.opensolaris.arc-caseid value=PSARC/2009/118 value=PSARC/2013/044
> +set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
> +dir path=usr/share/info
> +file path=usr/share/info/libconfig.info group=bin mode=0444 owner=root \
> +restart_fmri=svc:/application/texinfo-update:default facet.doc.info=true 
> +file path=/usr/include/libconfig.h
> +file path=usr/include/libconfig.h++
> +file path=usr/lib/libconfig.la 

You can use gmake sample-manifest to generate p5m file 

— 
Reply to this email directly or view it on GitHub .